docs/changehistory/NextVersion.md

```diff
@@ -174,11 +175,67 @@ Can also be attached as a map-layer:
 ### @itwin/core-common
 
 - [FontMap]($common) attempts to provide an in-memory cache mapping [FontId]($common)s to [Font](../learning/backend/Fonts.md) names. Use [IModelDb.fonts]($backend) instead.
+- Some types which are now more comprehensively exposed by backend's new `@itwin/ecschema-metadata` integration were made deprecated:
+  - [EntityMetaData]($common)
+  - [EntityMetaDataProps]($common)
+  - [CustomAttribute]($common)
+  - [PropertyMetaData]($common)
+  - [PropertyMetaDataProps]($common)
+
+| **Deprecated class from `@itwin/core-common`** | **Replacement class from `@itwin/ecschema-metadata`** |
+| ---------------------------------------------- | ----------------------------------------------------- |
+| `EntityMetaData`                               | Use `EntityClass` instead.                            |
+| `CustomAttribute`                              | Use `CustomAttribute` instead.                        |
+| `PropertyMetaData`                             | Use `Property` instead.                               |
 
 ### @itwin/core-backend
 
 - Use [IModelDb.fonts]($backend) instead of [IModelDb.fontMap]($backend).
-- Added dependency to ecschema-metadata and expose the metadata from various spots (IModelDb, Entity)
+- Added dependency to `@itwin/ecschema-metadata` and exposed the metadata from various spots (IModelDb, Entity).
+
+#### Deprecated metadata retrieval methods
+
+The `IModelDb.getMetaData(classFullName: string)` method has been deprecated in version 5.0. This method was used to get metadata for a class and would load the metadata from the iModel into the cache, if necessary.
+
+Similarly, other functions to retrieve metadata also have replacements:
+
+| **Deprecated from `@itwin/core-backend`** | **Replacement function**                                         | Usage                                                                        |
+| ----------------------------------------- | ---------------------------------------------------------------- | ---------------------------------------------------------------------------- |
+| `Element.getClassMetaData`                | Use `Element.getMetaData` from `@itwin/core-backend` instead.    | await entity.getMetaData()                                                   |
```

Comment:
wonder if we should wrap the usage in backticks


```suggestion
| `Element.getClassMetaData`                | Use `Element.getMetaData` from `@itwin/core-backend` instead.    | `await entity.getMetaData()`                                                   |
```

@ben-polinsky does our docs site support [inline code syntax highlighting](https://stackoverflow.com/questions/23226224/inline-code-syntax-highlighting-in-github-markdown)?

Reply:
I added this and the remaining backticks.
That stackoverflow mentioned extensions seems to be a Jekyll special syntax, which is not in the github flavored markdown spec: https://github.github.com/gfm/#code-span